recipes/seroba/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "seroba" %}
+{% set version = "1.0.0" %}
+{% set md5sum = "fcd6802070aa90d3f4810bb72a23cdce" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/sanger-pathogens/{{ name }}/archive/v{{ version }}.tar.gz
+  md5: {{ md5sum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
Oh, and the build requirements are only
```yaml
  build:
    - python
    - setuptools
```
, nothing else is needed of the build itself. (Ok, apart from `wget` -- but only because it is used in `build.sh` which could be removed when the `kmc` recipe is updated.)